openmetadata-ui/src/main/resources/ui/cypress/common/common.js

```diff
@@ -403,12 +406,23 @@ export const searchEntity = (term, suggestionOverly = true) => {
 
 export const visitEntityDetailsPage = (term, serviceName, entity) => {
   interceptURL('GET', '/api/v1/*/name/*', 'getEntityDetails');
-  interceptURL('GET', '/api/v1/search/*', 'explorePageSearch');
+  interceptURL(
+    'GET',
+    `/api/v1/search/query?q=*&from=*&size=*&index=${SEARCH_INDEX[entity]}`,
+    'explorePageTabSearch'
+  );
+  interceptURL(
+    'GET',
+    `/api/v1/search/suggest?q=*&index=dashboard_search_index,table_search_index,topic_search_index,pipeline_search_index,mlmodel_search_index`,
+    'searchQuery'
```

Comment:
Can we use the `SEARCH_INDEX` for the index value?

Reply:
it's fine in my opinion as we need here all the index to make URL and not any particular index or dynamic index